docs/excel/custom-functions-architecture.md

```diff
@@ -0,0 +1,42 @@
+---
+ms.date: 03/06/2019
+description: Learn about Excel custom functions' runtime. 
+title: Custom functions architecture (preview)
+localization_priority: Priority
+---
+# Custom functions architecture
+
+Custom function use a new JavaScript runtime that differs from the browser-based JavaScript engine which powers most other parts of your add-in.
```

Comment:
Would be good to add a stronger "why keep reading" or intent to the first sentence. 
- Should mention there are some  that get talked about in this article. (like you can't load any DOM usage libraries and you can't directly communicate with task pane code, etc...)
- I wouldn't call it a "new" runtime. It's just a separate runtime for reasons outlined below.